vagrant/provisioning/roles/activemq/tasks/main.yml

```diff
@@ -12,18 +12,42 @@
     state: present
     group: activemq
 
+- name: Create ActiveMQ directories
+  become: yes
+  file:
+    path: "{{ item }}"
+    state: directory
+    mode: "u+rwx,g+rx,o+rx"
```

Comment:
Why this change versus the current one?

Reply:
The directories were not created at first it had some permission issues and kept failing again and again. First i resolved this by creating the folders using the provision.sh file but after that I jus thought I could add an ansile task instead of using the .sh file